components/button/stories/button.stories.js

```diff
@@ -289,7 +308,7 @@ WithWrapping.parameters = {
 
 export const DisableWrapping = TextWrapTemplate.bind({});
 DisableWrapping.tags = ["!dev"];
-DisableWrapping.storyName = "Disable label wrap";
+DisableWrapping.storyName = "Text overflow disabled";
```

Comment:
Something about the previous display name kept tripping me up. I changed it to "Text overflow disabled" to sort of pair with the "Text overflow" story (to keep both examples together on the docs page).